index.js

```diff
@@ -21,6 +23,30 @@ const transformBabel = src => {
   return result;
 };
 
+
+const transformTs = (src, path) => {
```

Comment:
what about https://github.com/kulshekhar/ts-jest

maybe we can leverage that, instead of introducing own transform logic ?